### PR TITLE
chore(wmg+cellguide): remove calls to datasets and collections in discover api

### PR DESCRIPTION
**Pull request by atarashansky**
## Reason for Change

- Removed dependency of data viz apps on discover API. 

## Changes

- Updated source data generator in cellguide and dataset metadata generator in WMG along with tests

## Testing steps

- Tested with rdev pipeline runs
- Tests pass

### CODE REVIEW

File: backend/common/census_cube/data/constants.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "EFO:0010010": "CEL-seq2",
 }
 
-CENSUS_CUBE_PINNED_SCHEMA_VERSION = "5.0.0"
+CENSUS_CUBE_PINNED_SCHEMA_VERSION = "5.1.0"
```
Comment (Bento007) [inline backend/common/census_cube/data/constants.py:17]: why is this value hardcoded, shouldn’t it use the implied schema version from census?

File: backend/cellguide/pipeline/source_collections/source_collections_generator.py [hunk 2]
```diff
@@ -25,57 +46,35 @@ def generate_source_collections_data(all_cell_type_ids_in_corpus: list[str]) ->
...
         # Generate a mapping from collection id to SourceCollectionsData
         collections_to_source_data = {}
-        for dataset_id in dataset_ids:
-            dataset = datasets_dict[dataset_id]
-            collection_id = dataset["collection_id"]
 
-            # If we don't have any source data on this collection yet, create a new SourceCollectionsData object
+        for i in range(len(datasets_metadata_df)):
+            dataset_id = datasets_metadata_df.iloc[i]["dataset_id"]
+            if dataset_id not in dataset_ids:
```
Comment (Bento007) [inline backend/cellguide/pipeline/source_collections/source_collections_generator.py:61]: why would this happen? Add a comment please.

File: backend/wmg/pipeline/constants.py [hunk 3]
```diff
@@ -119,3 +119,23 @@
...
+ORGANISM_INFO = [
+    {"label": "homo_sapiens", "id": "NCBITaxon:9606"},
+    {"label": "mus_musculus", "id": "NCBITaxon:10090"},
+]
+
+
+class CensusParameters:
+    census_version = "latest"
+
+    def value_filter(organism: str) -> str:
```
Comment (Bento007) [inline backend/wmg/pipeline/constants.py:133]:
```suggestion
    @staticmethod
    def value_filter(organism: str) -> str:
```

File: backend/wmg/pipeline/constants.py [hunk 4]
```diff
@@ -119,3 +119,23 @@
...
 ]
+
+
+ORGANISM_INFO = [
+    {"label": "homo_sapiens", "id": "NCBITaxon:9606"},
+    {"label": "mus_musculus", "id": "NCBITaxon:10090"},
+]
+
+
+class CensusParameters:
```
Comment (Bento007) [inline backend/wmg/pipeline/constants.py:130]: What was your reasoning for making this a class?

**Review by Bento007 — CHANGES_REQUESTED** [review]
I see you are using asserts in the code. If this code is run with python optimizations the assert will be ignored.

File: backend/wmg/pipeline/constants.py (hunk 4 above)
Comment (atarashansky) [reply backend/wmg/pipeline/constants.py:130]: Easy to patch for testing.

**Review by Bento007 — APPROVED** [review]

**Pull request merged**